docs/works/30-issue.md

```
@@ -0,0 +1,26 @@
+너는 시니어 풀스택 엔지니어 + 아키텍트 역할이다.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)